certbot/colored_logging.py

```diff
@@ -1,45 +0,0 @@
-"""A formatter and StreamHandler for colorizing logging output."""
-import logging
-import sys
-
-from certbot import util
-
-
-class StreamHandler(logging.StreamHandler):
```

Comment:
In #4446, I define two more of these simple handler classes. Would you like to see them in their own file as well? If so, I'll make this change and fix up the other PR too.